src/utils/purge.js

```diff
@@ -3,12 +3,30 @@
 const { FASTLY_API_URL } = require('./constants');
 
 /**
- * Make a purge request to Fastly
+ * Make a purge request to Fastly via simple PURGE HTTP method.
  * @param {string} url
  * @returns {Promise<Object>} An object containing the request data
  * @throws {Error} Throws an error if the request was not successful
  */
-module.exports = url => {
+const purgeUnauthenticated = function (url) {
+  return fetch(url, { method: 'PURGE' })
+    .then(res => res.json())
+    .then(res => {
+      res.url = url;
+      return res;
+    })
+    .catch(e => {
```

Comment:
This is not necessary since you are not doing anything with the error itself, so the is not difference between doing this and not doing it.
```suggestion
```

Reply:
Yeah good point-- I copied the handling from the existing config, I'll update both!

Reply:
How's this?
https://github.com/clay/cache-clear/pull/9/commits/6c701f696e5e2c4879085632cd32e90d7b5fb403